src/components/MAPI/organizations/AccessControl/index.tsx

```diff
@@ -70,7 +71,12 @@ const AccessControl: React.FC<IAccessControlProps> = ({
     names: string[]
   ) => {
     try {
-      if (!activeRole || !data) return Promise.resolve();
+      if (!activeRole || !data) return Promise.resolve([]);
```

Comment:
This was returning `Promise<void>` - changing this to resolve to the same type so the case for a `void` value would not be necessary